common/src/main/java/whocraft/tardis_refined/client/renderer/blockentity/console/GlobalConsoleRenderer.java

```diff
@@ -47,21 +44,22 @@ public void render(GlobalConsoleBlockEntity blockEntity, float partialTick, Pose
         }
         poseStack.popPose();
 
-        if (theme == ConsoleTheme.CRYSTAL) {
+        if (theme == ConsoleTheme.CRYSTAL.getId()) {
             renderHoloShell(crystalHolo,270, blockEntity, poseStack, bufferSource, packedLight, crystalHoloColor);
         }
 
-        if (theme == ConsoleTheme.INITIATIVE) {
+        if (theme == ConsoleTheme.INITIATIVE.getId()) {
             renderHoloShell(initiativeHolo, -30 + 180, blockEntity, poseStack, bufferSource, packedLight, initiativeHoloColor);
         }
     }
 
     private void renderHoloShell(Vec3 offset, int rotation, GlobalConsoleBlockEntity blockEntity, PoseStack poseStack, MultiBufferSource bufferSource, int packedLight, Vec3 color) {
-        if (blockEntity.getLevel().random.nextInt(20) != 0) {
+       //TODO Implement
```

Comment:
Needs to use the AestheticsHandler potentially?

Reply:
Yes, just trying to think of a good solution for one - but I think I have one that's kinda horrible